README.md

```diff
@@ -1,62 +1,46 @@
+React - Redux Project
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Bookstore
 
+> "Bookstore"
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- React
+- Redux
+- CSS
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
```

Comment:
- [ ] [Optional] It would be nice to include instructions for cloning, installing and starting the development server in the getting started section of your README.md file.

```
git clone <git@github.com:Bushido-brown/Bookstore.git>
cd Bookstore
npm install
npm start
```